src/main/java/gov/nasa/cumulus/metadata/aggregator/processor/FootprintProcessor.java

```diff
@@ -20,11 +20,7 @@
 import java.math.BigInteger;
 import java.nio.charset.Charset;
 import java.nio.file.Paths;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Collection;
-import java.util.ArrayList;
+import java.util.*;
 
```

Comment:
I assume this is intellij or another IDE auto "optimizing" imports, but do we want to blanket import java.util.*? I know Nga used to prefer avoiding blanket imports like that, what do you think @skorper ?

Reply:
Great point @jocelynmallon! I agree, we should not use any * imports if we can avoid it.

Reply:
Thanks for the input, @jocelynmallon.
btw, you are right,  intelliJ removed the specific import and I overlooked it.